### ISSUE TITLE
Sort resource names in unmanaged resources list

### ISSUE THREAD

**Issue by adamdonahue** [cloudskiff/driftctl#321]
**Description**
Sort resource names in unmanaged resources list.  They appear to be unsorted at present, or sorted in search order?

**Example**
Here's what it looks like now, for example:

```
Found unmanaged resources:
  aws_ecr_repository:
    - s-dash/sample-3
    - s-dash/sample-data
    - s-dash/sample-1
    - s-dash/post-migration-app
    - plotly/reloader
    - s-dash/stag01
```
Perhaps this is only relevant to `aws_ecr_repository` objects.

**Comment by eliecharra** [comment]
Hi @adamdonahue thanks for your report and your interest in driftctl 🙏🏻 

As we run scan in parallel, the resources ordering is unpredictable. 

We may introduce a default sorting method based on resources ID's, and in some case maybe this logic needs to be overridden to use another field than ID for sorting, for example the same as you said.

To be clear, here you only expect a reproductible order or a alpha-numeric sort ? What's the most relevant for you ?

**Comment by adamdonahue** [comment]
I'm not talking about the order in which resources are processed.  Just the output.  Basic map/reduce pattern, essentially.  This is a really minor request though.

**Comment by adamdonahue** [comment]
I didn't dig into the code too deeply but if this would impact performance (if, for example, you streaming and flushing output as you go), then I'd argue it's very low priority.

**Comment by adamdonahue** [comment]
I noticed I didn't answer your question!  Just an alphanumeric sort -- makes it easier to scan and fine objects that need to be reconciled, but again, low priority.

**Comment by sundowndev** [comment]
Closing as it has been fixed in v0.8.0.
